src/datastore/index.js

```diff
@@ -25,7 +25,7 @@ import pDefer from 'p-defer'
 
 const NAMESPACE_SCHEMAS = /** @type {const} */ ({
   data: ['observation'],
-  config: ['preset', 'field'],
+  config: ['preset', 'field', 'project'],
   auth: [],
```

Comment:
I think there should be a comment here explaining what this is used for. took me a bit to figure out I needed to update it to add the project schema